Documentation/how-to-build-WebAssembly.md

```diff
@@ -0,0 +1,20 @@
+# Build WebAssembly #
+Currently, building WebAssembly is only possible on Windows.
```

Comment:
It maybe also have something like "current status" here - what is expected to work. This has been a frequent FAQ for other targets: people try to throw something really complex onto the compiler and it blows up.

Reply:
I'll open a high-level WebAssembly support issue to communicate things like that.